manifests/u/UnlimitedBacon/STL-Thumb/0.4.0/UnlimitedBacon.STL-Thumb.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: UnlimitedBacon.STL-Thumb
+PackageVersion: 0.4.0
+Installers:
+- Architecture: x86

```

I believe this installer works on x64 also
```suggestion
- Architecture: neutral
```

---

The git page say x64
https://github.com/unlimitedbacon/stl-thumb#windows